frontend/eda/views/RuleAudit/hooks/useRuleAuditActionsColumns.tsx

```diff
@@ -10,11 +12,50 @@ export function useRuleAuditActionsColumns() {
       {
         header: t('Name'),
         cell: (ruleAuditAction) => <TextCell text={ruleAuditAction?.name} />,
+        defaultSort: true,
+        card: 'name',
+        list: 'name',
+      },
+      {
+        header: t('Name'),
```

Comment:
@lgalis Do we need an extra column, can we just add a link to the name if the ruleAuditAction?.url? exists.